server/src/main/java/org/elasticsearch/common/cache/Cache.java

```diff
@@ -650,6 +650,36 @@ public void remove() {
         };
     }
 
+    /**
+     * An LRU sequencing of the entries in the cache. This sequence is not protected from mutations
+     * to the cache (except for {@link Iterator#remove()}. The result of iteration under any other mutation is
+     * undefined.
+     *
+     * @return an LRU-ordered {@link Iterable} over the entries in the cache
+     */
+    public Iterable<Tuple<K,V>> entries() {
```

Comment:
Just a drive by comment @tlrx this is still unused and can probably go away, just in case you missed it :)

I'll take a proper look at this PR in general now

Reply:
> Just a drive by comment @tlrx this is still unused and can probably go away, just in case you missed it :)

Thanks! I'll remove it.


> I'll take a proper look at this PR in general now

I was about to ping you once Henning approved the direction :)

Reply:
I removed the unused `entries()` method in 6b238d6e76b0ad8445feae40ef7076611d88a5c8